standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreChecker.java

```diff
@@ -303,56 +308,103 @@ void checkTable(Table table, PartitionIterable parts, byte[] filterExp, CheckRes
     if (tablePath == null) {
       return;
     }
-    FileSystem fs = tablePath.getFileSystem(conf);
-    if (!fs.exists(tablePath)) {
+    final FileSystem[] fs = {tablePath.getFileSystem(conf)};
```

Comment:
Is this just for tricking the `final` to accept changes?
Wouldn't be better for readability to use different variables instead?